Explorer/Controller/ViewController.swift

```diff
@@ -23,11 +23,16 @@ class ViewController: UIViewController {
         return tableView
     }()
     
+    let createLogViewController = CreateLogViewController()
```

Comment:
nit: Doesn't seem like you need to store a reference here, you can just call `present()` on `CreateLogViewController()` directly.